tests/scripts/command-from-callback-test.sh

```diff
@@ -54,6 +54,8 @@ timeout 5s ./simulated-redis.pl -p 7402 -d --sigcont $syncpid2 <<'EOF' &
 EXPECT CONNECT
 EXPECT ["GET", "fee"]
 SEND -MOVED 12182 127.0.0.1:7401
```

Comment:
This slot in received -MOVED wasn't used before and it seems that this value is incorrect in the legacy testcase.
The key `fee` would be slot 8471 if I calculated correct, i.e. change to
`SEND -MOVED 8471 127.0.0.1:7401`

same issue for MOVED of `foo`